website/docs/r/route.html.markdown

```diff
@@ -89,3 +89,29 @@ will be exported as an attribute once the resource is created.
 
 - `create` - (Default `2 minutes`) Used for route creation
 - `delete` - (Default `5 minutes`) Used for route deletion
+
+## Import
+
+Individual routes can be imported using a pseudo ID defined as r-ROUTETABLEID_DESTINATION.
+
+For example, to import a route in a route table with ID rtb-656C65616E6F72 and an IPv4 CIDR destination of 10.42.0.0/16,
+the import ID would be as follows:
```

Comment:
Nitpick: We can remove the indirection of calling something the "import ID" and just show what we mean:

``````md
For example, to import a route in a route table with ID `rtb-656C65616E6F72` and an IPv4 CIDR destination of `10.42.0.0/16`:

```
$ terraform import aws_route.my_route rtb-656C65616E6F72_10.42.0.0/16
```

To import a route from route table `rtb-656C65616E6F72` with an IPv6 destination CIDR of `2620:0:2d0:200::8/125`:

```
$ terraform import aws_route.my_route rtb-656C65616E6F72_2620:0:2d0:200::8/125
```
``````